contracts/dispatcher/src/contract.rs

```diff
@@ -229,30 +232,34 @@ mod tests {
 
         do_instantiate(deps.as_mut());
 
-        let unauth_info = mock_info("anyone", &coins(2, "token"));
-        let msg = ExecuteMsg::Config { cadence_hours: 20 };
-        let res = execute(deps.as_mut(), mock_env(), unauth_info, msg);
-        match res {
-            Err(ContractError::Unauthorized(..)) => {}
-            _ => panic!("Must return unauthorized error"),
-        }
-
-        let auth_info = mock_info("creator", &coins(2, "token"));
-        let msg = ExecuteMsg::Config { cadence_hours: 12 };
-        let _res = execute(deps.as_mut(), mock_env(), auth_info, msg).unwrap();
+        let _auth_info = mock_info("creator", &coins(2, "token"));
+        drop(
```

Comment:
[everywhere] let's not use `drop` - the message it sends differs from our intention. Instead check the result for everything you might come up with, e.g. no messages, etc...